microsoft.ui.xaml/uielement_keyboardacceleratorplacementtarget.md

```diff
@@ -15,11 +15,11 @@ Gets or sets a value that indicates the control [tooltip](../microsoft.ui.xaml.c
 
 ## -property-value
 
-The control tooltip that displays the accelerator key combination.
+The control tooltip displays the accelerator key combination.
```

Comment:
This line can probably stay as-is or update "that" to "which". The 'value' is the control that displays the key combo.

Reply:
Changed 'the control tooltip that displays'